e2e/playwright/navigation/src/tests/sidebar.spec.ts

```diff
@@ -0,0 +1,126 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, APP_ROUTES, getUserState, SIDEBAR_LABELS, test } from '@alfresco/playwright-shared';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('Sidebar', () => {
+  const apiClientFactory = new ApiClientFactory();
+
+  test.beforeAll(async () => {
+    await apiClientFactory.setUpAcaBackend('hruser');
+  });
+
+  test('[C289902] navigate to Favorite Libraries', async ({ personalFiles, favoritePage }) => {
```

Comment:
the sidebar is a single component that is part of the application layout, you don't need testing each URL to have the same behaviour in this case. Just test the "Personal Files" route and you can reduce the tests

Reply:
I have remove such few test case.